src/components/dropdown/CustomRender.tsx

```diff
@@ -125,9 +125,11 @@ export const DropdownIndicator = (props: any) => {
     </components.DropdownIndicator>;
 };
 
-export const ClearIndicator = (props: any) =>
+export const ClearIndicator = (props: any) => 
   <components.ClearIndicator {...props}>
-    <Icon iconName="cross-round"/>
+    <div className="tk-select__close-icon" onKeyPress={props.clearValue} tabIndex={0}>
```

Comment:
What is the use of "tk-select__close-icon" ?

Reply:
These styles come from UItoolkit-styles.

Reply:
But I can't find it anywhere on UI Toolkit-styles
![image](https://user-images.githubusercontent.com/56824367/130030712-814e2b80-f13c-4b49-a4ed-9fccdeb07519.png)
![image](https://user-images.githubusercontent.com/56824367/130030767-1e7b12f9-7d46-4d38-80be-c6a17f6d51e5.png)

Reply:
Now I see https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles/pull/125 open so I understand :)